heron/tools/tracker/src/python/tracker.py

```diff
@@ -100,11 +103,15 @@ def get_topology(
                   and t.cluster == cluster
                   and (not role or t.execution_state.role == role)
                   and t.environ == environ]
-    if len(topologies) != 1:
+    if len(topologies) == 0:
```

Comment:
as this is a behaviour change, is it known to rectify anything? now it warns if there is more than one topology, but not I wonder if that is a but. I don't recall if this would cause issues (without reading around a lot), so not sure it should be included unless it is a known improvement

Reply:
okay. I'll erase it.